src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -14279,7 +14279,10 @@ void Unit::UpdateSpeed(UnitMoveType mtype, bool forced)
                     stack_bonus     = GetTotalAuraMultiplier(SPELL_AURA_MOD_MOUNTED_FLIGHT_SPEED_ALWAYS);
                 }
                 else             // Use not mount (shapeshift for example) auras (should stack)
+                {
                     main_speed_mod  = GetTotalAuraModifier(SPELL_AURA_MOD_INCREASE_FLIGHT_SPEED) + GetTotalAuraModifier(SPELL_AURA_MOD_INCREASE_VEHICLE_FLIGHT_SPEED);
+                    stack_bonus = GetTotalAuraMultiplier(SPELL_AURA_MOD_MOUNTED_FLIGHT_SPEED_ALWAYS);
```

Comment:
This change is not needed; This adds mount speed increases (`SPELL_AURA_MOD_MOUNTED_FLIGHT_SPEED_ALWAYS`) to affect non-mounts (`SPELL_AURA_MOD_INCREASE_VEHICLE_FLIGHT_SPEED`)

e.g. carrot and spurs will now boost flight form

Reply:
Modifying the dbc of 48383 to apply below auras should work
```
SPELL_AURA_MOD_MOUNTED_FLIGHT_SPEED_ALWAYS
SPELL_AURA_MOD_MOUNTED_SPEED_ALWAYS
```

Reply:
> This change is not needed; This adds mount speed increases (`SPELL_AURA_MOD_MOUNTED_FLIGHT_SPEED_ALWAYS`) to affect non-mounts (`SPELL_AURA_MOD_INCREASE_VEHICLE_FLIGHT_SPEED`)
> 
> e.g. carrot and spurs will now boost flight form

application is correct

Reply:
So this should be updated in spell info correction?

Reply:
y below should work. So it behaves more like mithril spurs 

`SPELL_RIDING_CROP_EFFECT = 48383,` from spell_item.cpp
```
diff --git a/src/server/game/Spells/SpellInfoCorrections.cpp b/src/server/game/Spells/SpellInfoCorrections.cpp
index ca47ac9fe1..66e671e7fb 100644
--- a/src/server/game/Spells/SpellInfoCorrections.cpp
+++ b/src/server/game/Spells/SpellInfoCorrections.cpp
@@ -4852,2 +4852,9 @@ void SpellMgr::LoadSpellInfoCorrections()
 
+    // Mount Speed (Riding Crop effect)
+    ApplySpellFix({ 48383 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->Effects[EFFECT_0].ApplyAuraName = SPELL_AURA_MOD_MOUNTED_SPEED_ALWAYS;
+        spellInfo->Effects[EFFECT_1].ApplyAuraName = SPELL_AURA_MOD_FLIGHT_SPEED_NOT_STACK;
+    });
+
```

spellwork
![riding_crop_spurs_dbc](https://github.com/user-attachments/assets/0dc9af8c-e74c-4def-a923-ff3cd9ebc5c6)

Reply:
I understand, but it doesn't solve the problem, because it's not in that spell

Reply:
> So this should be updated in spell info correction?

there was a problem, the change I made is stack with crusader aura